webpack.config.js

```diff
@@ -208,6 +208,10 @@ if ( calypsoEnv === 'production' ) {
 	) );
 }
 
+if ( ! config.isEnabled( 'desktop' ) ) {
+	webpackConfig.plugins.push( new webpack.NormalModuleReplacementPlugin( /^lib\/desktop$/, 'lodash/noop' ) );
```

Comment:
I would rather explore other approaches. It would be nice to have Desktop specific boot file which would be the only place where those kind of logic is initialized.